bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "collabosphere",
+  "description": "Collabosphere Instructure Canvas LTI Tools",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ets-berkeley-edu/collabosphere.git"
+  },
+  "dependencies": {
+    "angular": "latest",
```

Comment:
Similarly, can you provide an example Apache config? I assume it'll be a straightforward:
 - ProxyPass /api
 - DocumentRoot ~/path/to/collabosphere/public